sorc/grid_tools.fd/filter_topo.fd/filter_topo.F90

```diff
@@ -1295,6 +1334,8 @@ subroutine two_delta_filter(is, ie, js, je, isd, ied, jsd, jed, npx, npy, ntiles
     integer:: i,j, nt, t
     integer:: is1, ie2, js1, je2
 
+    a2 = 0.
```

Comment:
@RatkoVasic-NOAA I might add a comment here about this being a simple fix and your suggestion for a better fix.

Reply:
Let's hold on for the moment, I'm looking into better solution.

Reply:
Great. By the way, the random behavior seems to have disappeared. I tested 06cc530 using Intel/GNU, Release/Debug compilations. My canned case gave identical results for all four tests. So I think you found all the problem arrays.